spec/requests/vendors_spec.rb

```diff
@@ -1,25 +1,36 @@
 require "rails_helper"
 
 RSpec.describe "/vendors" do
-  let!(:vendor) { create(:vendor) }
+  let!(:vendor1) { create(:vendor) }
+  let!(:vendor2) { create(:vendor) }
 
   before { sign_in(create(:user)) }
 
   describe "GET /vendors" do
     it "renders a list of vendors" do
       get "/vendors"
       expect(response).to be_successful
-      expect(response.body).to include(vendor.folio_id)
-      expect(response.body).not_to include(vendor.folio_code)
+      body = response.body
+
+      expect(body).to include("Folio's Vendor Status</th>")
+      expect(body).to include("Last Scheduled Job</th>")
+
+      expect(body).to include("/vendors/#{vendor1.folio_id}\">#{vendor1.name}</a>")
+      expect(body).to include("ID: #{vendor1.folio_id}")
+      expect(body).not_to include(vendor1.folio_code)
+
+      expect(body).to include("/vendors/#{vendor2.folio_id}\">#{vendor2.name}</a>")
```

Comment:
I wonder if we can avoid coupling to specific HTML elements and get away with testing on the strings?

Reply:
I wanted to make sure we had the links.

Reply:
Okie dokie, no problem.